src/java/org/apache/cassandra/tools/nodetool/Stop.java

```diff
@@ -48,4 +48,4 @@ public void execute(NodeProbe probe)
         else
             probe.stop(compactionType.name());
     }
-}
\ No newline at end of file
+}
```

Comment:
I think this is an unwanted change

Reply:
Yes, it didn't change anything. Removed and re-added  "}"
Because of the merge.